app/views/api/asciicasts/show.html.slim

```diff
@@ -19,6 +19,14 @@ javascript:
       if (e.data == 'asciicast:play' || e.data[0] == 'asciicast:play') {
         player.play();
       }
+
+      // reveal.js support
+
+      if (e.data == 'slide:start') {
```

Comment:
This is really cool; the question is whether you want to enable it unconditionally, though. Note that reveal.js *always* posts this message, not just when `data-autoplay` is set on the parent iframe.